### PR TITLE
SMD-173: enforce role membership

### PR DESCRIPTION
**Pull request by gidsg**
### Change description
Enforce user role membership to access upload route
- [x] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
* In docker runner: user should not be able to access upload route unless they are given the role. Ensure you have latest versions of authenticator and account-store for this to work
* I can add people to the role to test being able to access once role is added
 
### Screenshots of UI changes (if applicable)
Screenshot 2023-10-13 at 15 59 44

### CODE REVIEW

File: app/main/routes.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def index():
 
 
 @bp.route("/upload", methods=["GET", "POST"])
-@login_required(return_app=SupportedApp.POST_AWARD_SUBMIT)
+@login_required(return_app=SupportedApp.POST_AWARD_SUBMIT, roles_required=["TF_MONITORING_RETURN_SUBMITTER"])
```
Comment (cyrusdobbs) [inline app/main/routes.py:24]: Could `TF_MONITORING_RETURN_SUBMITTER` be defined as a constant and then referenced throughout the application

File: config/envs/development.py [hunk 2]
```diff
@@ -21,4 +21,8 @@ class DevelopmentConfig(DefaultConfig):
             ("Sunderland City Council", "Worcester City Council"),
             ("Sunderland City Centre", "Blackfriars - Northern City Centre", "Worcester"),
         ),
+        "test.communities.gov.uk": (
+            ("Sunderland City Council", "Worcester City Council"),
+            ("Sunderland City Centre", "Blackfriars - Northern City Centre", "Worcester"),
+        ),
```
Comment (cyrusdobbs) [inline config/envs/development.py:27]: How come we need this?

**Review by cyrusdobbs — CHANGES_REQUESTED** [review]
Nice :) couple of comments

**Comment by cyrusdobbs** [comment]
Can this be squashed on merge 👍

File: config/envs/development.py (hunk 2 above)
Comment (gidsg) [reply config/envs/development.py:27]: This is the domain used by the Test AD, so it's a convenience for users that have Test AD accounts.

File: app/main/routes.py (hunk 1 above)
Comment (gidsg) [reply app/main/routes.py]: OK have done this now in: 8717627

**Review by cyrusdobbs — APPROVED** [review]
LGTM. Just a reminder to squash 👍

**Pull request merged**